ImagePickerSheetController/ImagePickerSheetController/ImagePickerSheetController.swift

```diff
@@ -65,6 +65,9 @@ public class ImagePickerSheetController: UIViewController, UITableViewDataSource
         }
     }
     
+    /// Maximum selection of images, if maximumSelection = 1 (previous will be unselected), if limit > 1 = first will be unselected.
+    public var maximumSelection = 0
```

Comment:
Would you mind making this property optional. Then it's clear what the default value is.